resources/kcp/charts/kyma-environment-broker/templates/subscription-cleanup-job.yaml

```diff
@@ -16,7 +16,7 @@ spec:
             - image: "{{ .Values.global.images.containerRegistry.path }}/{{ .Values.global.images.kyma_environments_subscription_cleanup_job.dir }}kyma-environment-subscription-cleanup-job:{{ .Values.global.images.kyma_environments_subscription_cleanup_job.version }}"
               name: subscription-cleanup-job
               command:
-                - "/bin/subscriptioncleanup"
+                - /bin/main
```

Comment:
Herę you will change the command for environment subscription cleanup job, you change to docker image `Dockerfile.job` but you do not change the image reference. It will still point to `PR-2554` and job will fail.